sampler.go

```diff
@@ -418,14 +418,14 @@ func NewRemotelyControlledSampler(
 		logger:                  NullLogger,
 		metrics:                 NewMetrics(NullStatsReporter, nil),
 		samplingRefreshInterval: defaultSamplingRefreshInterval,
-		hostPort:                defaultSamplingServerHostPort,
+		samplingServerURL:       defaultSamplingServerURL,
 		sampler:                 initialSampler,
 		maxOperations:           defaultMaxOperations,
 	}
 
 	sampler.applyOptions(options...)
 	sampler.timer = time.NewTicker(sampler.samplingRefreshInterval)
-	sampler.manager = &httpSamplingManager{serverURL: "http://" + sampler.hostPort}
+	sampler.manager = &httpSamplingManager{serverURL: "http://" + sampler.samplingServerURL}
```

Comment:
is this technically a URL if it does not have `http://`?  Maybe it should be part of the value to begin with.